assets/app/view/game/actionable.rb

```diff
@@ -95,6 +95,7 @@ def process_action(action)
 
         if game.finished
           @game_data[:result] = game.result
+          @game_data[:finished_at] = Time.now.to_i
```

Comment:
what is this? is this a one time set?

Reply:
Yes, a one-time set. This is to maintain the the game finish date, while allowing dev changes that cause the json to change. Otherwise, every game will appear like it finished when the json change is applied.

Reply:
This particular location is because of hot seat games.